_includes/sidenav.html

```diff
@@ -6,46 +6,22 @@
 <aside class="usa-layout-docs-sidenav desktop:grid-col-3 padding-bottom-4">
   <nav>
     <ul class="usa-sidenav">
+      <li class="usa-sidenav__item">
+        <a href="/about/">Learning about OpenACRs</a>
```

Comment:
@mgifford it looks like we should add `{{ site.baseurl }}` to the beginning of the href URL so the links work. Otherwise it tried the absolute link and fails to work in federalist. Can you try that?

Reply:
Wil do

Reply:
Changed